### ISSUE TITLE
Running pyicontract-lint without linting errors should notify success

### ISSUE THREAD

**Issue by fabiosangregorio** [Parquery/pyicontract-lint#23]
Hi @mristin , it's me again :)
Just an enhancement proposal: when I got the package to run I had no linting errors and therefore the package would not give any output, even in `--verbose` mode. I had to run it with `--verbose json` mode to get the following output:
```bash
> pyicontract-lint --format verbose telereddit  # no output
> pyicontract-lint --format json telereddit
[]
```

It would be great if we could get some sort of feedback on the successful run (at least in `--verbose` mode). I'm thinking of something along the line of:
```bash
> pyicontract-lint --format verbose telereddit
--------------------------------------------------------------------
Checked xx files. OK
```

**Comment by mristin** [comment]
Hi @fabiosangregorio ,
I added a very brief output ("no errors detected") in PR #25. Adding more information (like number of files inspected) would cause too many changes in the code which I prefer not to touch as long as it's not clear that there are real use cases. Please let me know if you have a case for a concrete nugget of information.

Let's wait a couple of days till no new feature or bug fix is added before releasing the next version. I expected that you might come up with more issues in the meanwhile :-).

**Comment by fabiosangregorio** [comment]
That's awesome! I'll let you know if I notice something :)

**Comment by mristin** [comment]
Hi @fabiosangregorio ,
I've just released the minor version 2.1.0 that includes all the recent features (including this one). Thanks again for your feedback!
